main/src/ca/uwaterloo/flix/api/lsp/provider/completion/Completion.scala

```diff
@@ -47,10 +47,11 @@ sealed trait Completion {
         sortText = Priority.normal(name),
         textEdit = TextEdit(context.range, s"$name "),
         kind = CompletionItemKind.Keyword)
-    case Completion.FieldCompletion(name) =>
+    case Completion.FieldCompletion(field, prefix) =>
```

Comment:
What is prefix and why is it needed?

Reply:
The way `FieldCompleter` is constructed is quite confusing. I'll remove anything not needed in `FieldCompletion` when I'm fixing `FieldCompleter`. Will be in next PR.

Reply:
Okay, but I don't like merging things that are halfway done, so I will hold you to your promise to fix it :)

Reply:
I Will, however, if you like, you Can just hold this PR until the other one is merged

Reply:
On the other hand, I just realized:

The prefix makes sure, that the "variable" defined as a record will be included, and not only the field.

```
def main(): Unit \ IO = 
    let p = { x = 1, y = 2 };
    let sum = p.x             <- The prefix is "p"
    println()
```